### PR TITLE
Use block syntax

### PR DESCRIPTION
**Pull request by AurelienFT**
DO NOT MERGE INTERNAL REVIEW PR.

### CODE REVIEW

**Review by damip — APPROVED** [review]

File: parser/src/ast/declarations.rs [hunk 1]
```diff
@@ -292,14 +292,14 @@ impl PartialEq for PublicInput {
 /// size `15`:
 ///
 /// ```airscript
-/// random_values:
+/// random_values {
```
Comment (bilboquet) [inline parser/src/ast/declarations.rs:295]: unclosed block

File: parser/src/ast/declarations.rs [hunk 2]
```diff
@@ -292,14 +292,14 @@ impl PartialEq for PublicInput {
...
 ///     rand: [15]
 /// ```
 ///
 /// A `random_values` declaration like the following however:
 ///
 /// ```airscript
-/// random_values:
+/// random_values {
```
Comment (bilboquet) [inline parser/src/ast/declarations.rs:303]: unclosed block

File: parser/src/ast/statement.rs [hunk 3]
```diff
@@ -97,7 +97,7 @@ pub struct Let {
     /// For example, given the following:
     ///
     /// ```airscript
-    /// integrity_constraints:
+    /// integrity_constraints {
```
Comment (bilboquet) [inline parser/src/ast/statement.rs:100]: unclosed

File: docs/src/description/variables.md [hunk 4]
```diff
@@ -43,7 +48,7 @@ Currently, it is not possible to:
 
     ```
     ...
-    boundary_constraints:
+    boundary_constraints {
```
Comment (bilboquet) [inline docs/src/description/variables.md:51]: unclosed

File: docs/src/description/variables.md [hunk 5]
```diff
@@ -57,7 +62,7 @@ Currently, it is not possible to:
 
     ```
     ...
-    integrity_constraints:
+    integrity_constraints {
```
Comment (bilboquet) [inline docs/src/description/variables.md:66]: unclosed

File: docs/src/description/declarations.md [hunk 6]
```diff
@@ -96,7 +98,7 @@ A `random_values` section contains declarations for random values provided by th
 The following is an example of a valid `random_values` source section:
 
 ```
-random_values:
+random_values {
```
Comment (bilboquet) [inline docs/src/description/declarations.md:102]: unclosed

File: docs/src/description/declarations.md [hunk 7]
```diff
@@ -105,7 +107,7 @@ In the above example, `rand` is a random value array of length `4`. Random value
 The following is an example of a valid `random_values` section with named bindings to particular random values and groups of random values:
 
 ```
-random_values:
+random_values {
```
Comment (bilboquet) [inline docs/src/description/declarations.md:112]: unclosed

File: docs/src/description/declarations.md [hunk 8]
```diff
@@ -47,7 +48,7 @@ identifier: [n]
 The following is an example of a valid `public_inputs` source section:
 
 ```
-public_inputs:
+public_inputs {
```
Comment (bilboquet) [inline docs/src/description/declarations.md:51]: unclosed

**Review by bilboquet — CHANGES_REQUESTED** [review]
Some blocks are not closed in documentation and comments.
I think I've spotted them all, see my comments.

**Comment by AurelienFT** [comment]
All comments has been addressed, I'm making a PR in the official repository

**Pull request closed without merging**